src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -1386,6 +1386,8 @@ private void AddLanguageBuffer()
                     bufferAdded(_window, new SubmissionBufferAddedEventArgs(buffer));
                 }
 
+                _window.LanguageBufferCounter++;
+
```

Comment:
Per offline conversation, this is exactly in the method that I mentioned above so it is good.  I was misguided when the CR view showed this under SubmitAsync.